editor/src/components/inspector/controls/pin-control.spec.browser2.tsx

```diff
@@ -0,0 +1,660 @@
+/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "makeTestCase"] }] */
+import {
+  BakedInStoryboardUID,
+  BakedInStoryboardVariableName,
+} from '../../../core/model/scene-utils'
+import { RegisteredCanvasStrategies } from '../../canvas/canvas-strategies/canvas-strategies'
+import type { EditorRenderResult } from '../../canvas/ui-jsx.test-utils'
+import {
+  TestAppUID,
+  TestSceneUID,
+  formatTestProjectCode,
+  getPrintedUiJsCodeWithoutUIDs,
+  renderTestEditorWithCode,
+} from '../../canvas/ui-jsx.test-utils'
+import * as EP from '../../../core/shared/element-path'
+import {
+  selectComponentsForTest,
+  setFeatureForBrowserTestsUseInDescribeBlockOnly,
+} from '../../../utils/utils.test-utils'
+import { mouseClickAtPoint } from '../../canvas/event-helpers.test-utils'
+import { getDomRectCenter } from '../../../core/shared/dom-utils'
+
+function makeTestProjectCode(componentInnards: string): string {
+  const code = `
+  import * as React from 'react'
+  import { Scene, Storyboard, View, Group, Rectangle } from 'utopia-api'
+
+
+  export var App = (props) => {
+    return (
+      ${componentInnards}
+    )
+  }
+
+  export var ${BakedInStoryboardVariableName} = (props) => {
+    return (
+      <Storyboard data-uid='${BakedInStoryboardUID}'>
+        <Scene
+          style={{ left: 0, top: 0, width: 400, height: 400 }}
+          data-uid='${TestSceneUID}'
+        >
+          <App
+            data-uid='${TestAppUID}'
+            style={{ position: 'absolute', bottom: 0, left: 0, right: 0, top: 0 }}
+          />
+        </Scene>
+      </Storyboard>
+    )
+  }
+`
+  return formatTestProjectCode(code)
+}
+
+function makeTestProjectCodeWithoutUIDs(componentInnards: string): string {
+  const code = `
+  import * as React from 'react'
+  import { Scene, Storyboard, View, Group, Rectangle } from 'utopia-api'
+
+
+  export var App = (props) => {
+    return (
+      ${componentInnards}
+    )
+  }
+
+  export var ${BakedInStoryboardVariableName} = (props) => {
+    return (
+      <Storyboard>
+        <Scene
+          style={{ left: 0, top: 0, width: 400, height: 400 }}
+        >
+          <App
+            style={{ position: 'absolute', bottom: 0, left: 0, right: 0, top: 0 }}
+          />
+        </Scene>
+      </Storyboard>
+    )
+  }
+`
+  return formatTestProjectCode(code)
+}
+
+interface TestCase {
+  baseProject: string
+  actionChange: (renderResult: EditorRenderResult) => Promise<void>
+  expectedProject: string
+}
+
+xdescribe('Group child constraints', () => {
```

Comment:
Was this meant to be left as `xdescribe`?

Reply:
It was an intentional change from this commit: https://github.com/concrete-utopia/utopia/pull/4439/commits/e5260e2be7e9dd6413f1df3e1962c3ffa2ba77dd

Reply:
yes I changed it to xdescribe along with replacing the actual pin click handler with a NO_OP. there's a detailed follow-up task in the paper doc how to re-enable the actual toggling behaviors, which are different for frame children, group children etc. 

on master the pin toggling is disabled because for groups it was broken.

this PR accidentally resurrected the old-world usePinToggling which is buggy so we decided to keep "the behavior" from master: disable them for now